packages/neuron-ui/src/components/SendFieldset/index.tsx

```diff
@@ -96,77 +106,55 @@ const SendFieldset = ({
-            type="button"
-            disabled={isAddOneBtnDisabled}
-            onClick={onOutputAdd}
-            aria-label={t('send.add-one')}
-            className={styles.iconBtn}
-          >
-            <img src={AddOutput} alt="Add Output" data-type="add" />
-          </button>
-        ) : null}
-      </div>
-
-      {locktimeAble ? (
-        <div className={styles.locktime} data-status={item.date ? 'set' : 'unset'}>
-          <img data-status="inactive" className={styles.icon} src={Calendar} alt="calendar" />
-          <img data-status="active" className={styles.icon} src={ActiveCalendar} alt="active-calendar" />
-          {item.date ? `${t('send.release-on')}: ${formatDate(new Date(+item.date))}` : null}
-          <button type="button" data-index={idx} data-type="set" onClick={onLocktimeClick}>
-            {item.date ? (
-              <>
-                <img data-status="inactive" className={styles.icon} src={Edit} alt="edit" />
-                <img data-status="active" className={styles.icon} src={ActiveEdit} alt="active-edit" />
-              </>
-            ) : (
-              t('send.set-locktime')
-            )}
-          </button>
+      {locktimeAble && (
+        <div className={styles.locktime}>
           {item.date ? (
-            <button type="button" data-index={idx} data-type="remove" onClick={onLocktimeClick}>
-              <img data-status="inactive" className={styles.icon} src={Trash} alt="trash" />
-              <img data-status="active" className={styles.icon} src={ActiveTrash} alt="active-trash" />
-            </button>
-          ) : null}
-          {item.date && (
-            <div className={styles.locktimeWarn}>
-              <Attention />
-              {t('send.locktime-warning')}
+            <div>
+              <div className={styles.content}>
+                <img className={styles.icon} src={TimeClock} alt="calendar" />
+                <p>{`${t('send.release-on')}: ${formatDate(new Date(+item.date))}`}</p>
+                <button type="button" onClick={onLocktimeClick}>
+                  <Trash data-index={idx} data-type="remove" />
+                </button>
+              </div>
+              <div className={styles.locktimeWarn}>
+                <Attention />
+                {t('send.locktime-warning')}
+              </div>
+            </div>
+          ) : (
+            <div className={styles.content}>
+              <img className={styles.icon} src={TimeClock} alt="calendar" />
+              <button type="button" data-index={idx} data-type="set" onClick={onLocktimeClick}>
+                {t('send.set-locktime')}
+              </button>
             </div>
           )}
         </div>
-      ) : null}
+      )}
+
+      <AlertDialog
+        show={showDeleteDialog}
+        title={t('send.remove-receiving-address')}
+        message={t('send.remove-receiving-address-msg')}
+        type="warning"
+        onCancel={() => setShowDeleteDialog(false)}
```

Comment:
Is it different from without `onCancel`? I see the default is `closeDialog`.

Reply:
AlertDialog doesn't have closeDialog.  it cannot be closed without onCancel.